node/rpc-api/src/lib.rs

```diff
@@ -333,6 +335,10 @@ pub mod state_api {
     pub type StateNetworkNameParams = ();
     pub type StateNetworkNameResult = String;
 
+    pub const STATE_START_TIME: &str = "Filecoin.StateStartTime";
+    pub type StateStartTimeParams = ();
+    pub type StateStartTimeResult = String;
```

Comment:
This RPC call returns a `String` to match the API in Lotus?

Reply:
Ah this should be part of common_api, i've moved it. Lotus returns a time.Time, i've refactored accordingly to return a OffsetDateTime.